pkg/prom/prom.go

```diff
@@ -384,6 +403,7 @@ func NewPrometheusClient(address string, config *PrometheusClientConfig) (promet
 		TLSHandshakeTimeout: config.TLSHandshakeTimeout,
 		TLSClientConfig: &tls.Config{
 			InsecureSkipVerify: config.TLSInsecureSkipVerify,
+			RootCAs:            tlsCaCert,
```

Comment:
Could you show some test results where you've run OpenCost with `ENABLE_KUBE_RBAC_PROXY=true` and `ENABLE_KUBE_RBAC_PROXY=false`. Just want to make sure a `nil` tlsCaCert is ok here.

Reply:
sure let me update the pr description for it